draft-rfcs/working-group-terminology.md

```diff
@@ -2,96 +2,274 @@
+how to create a group.
 
-Working groups are a way of finding new domains that would benefit from continual development. Though it is not required, if a working group has demonstrated consistent productivity over a significant period time, it can become a Rust team.
+It is recommended for the working group to create a `wg-{name}`/`project-{name}`
+repository under the `rust-lang` organisation using the [project group
+template], and making any relevant changes and personalisation.
 
-- Creation of a Working Group is approved by the Core team.
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[project group template]: https://github.com/rust-lang/project-group-template
 
-### Project Groups
+## Working Groups
 
-A Project Group is a group of people working on a particular project or responsibilities at the behest of an official Rust team. Examples of this would include FFI Unwind, Triage, or The Rust Survey.
+Working groups are a way of finding new domains that would benefit from
+continual development, that are not covered by an existing Rust team. Examples
+of this include [Embedded][embedded-wg], [WebAssembly][wasm-wg], and
+[Game Development][gamedev-wg].
 
-- Creation of a Project Group is approved by the related team.
+[embedded-wg]: https://github.com/rust-embedded/wg
+[wasm-wg]: https://github.com/rustwasm/team
+[gamedev-wg]: https://github.com/rust-gamedev/wg
 
-### Community Groups
+### Working Group Creation
 
-Community Groups are groups of individuals who want to work and collaborate on a specific topic, without going through the formal process of a working group. Community groups are free to adopt the structure of working or projects groups or create their own structure.
+Creation of a working group is approved by the core team. Typically this has
+been done by the core team agreeing to approve the creation of new working
+groups and having a period of time soliciting applications from the community,
+and then approving a subset of those applications.
 
-- Community groups are not required to follow any of the guidelines mentioned above.
-- As such community groups are not officially endorsed by the Rust project.
+> **DRAFT NOTE** Should this application come in the form of an RFC? My
+> inclination is yes, however it could just create more churn and drama than
+> needed. Posting in a thread on internals as was done previously might
+> be enough.
 
-# Drawbacks
+#### Application Checklist
+
+This not meant to be formal list of questions to be answer, however the
+application should try to address these points.
+
+- What value do you see your group bringing to the organisation?
+- Why should the project your group?
+- What are the goals of your group?
+- What are explicitly non-goals of your group?
+
+These first four points should also serve as the basis for the charter for the
+working group.
 
-[drawbacks]: #drawbacks
+- Is your group long-running or temporary?
+  - If it is temporary, how long do you see it running for?
```

Comment:
If the answer is "temporary", I suspect that might be a sign that the group would be better organized as a project group. Is that worth mentioning here? (I don't mean to imply that working groups should never be temporary, just that that's probably rarer than temporary project groups.)

Reply:
Well no, because a project group would be part of a team, and are in a way managed by the team, where as a working group is usually self managed.